lib/macos_ui.dart

```diff
@@ -1,7 +1,5 @@
 library macos_ui;
 
-export 'package:flutter/cupertino.dart'
```

Comment:
This removal reminds me that we really shouldn't be exporting any non-macos_ui libraries in this file, it messes with the generation of documentation. If we need the convenience of exporting flutter libraries in one file, we should have a separate file specifically for this that is not exported in `lib/macos_ui.dart`. Perhaps a `lib/src/library.dart`.

Reply:
I will tinker with this now

Reply:
Made the change, running dartdoc now to see the results